planning/rtc_interface/src/rtc_interface.cpp

```diff
@@ -0,0 +1,193 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rtc_interface/rtc_interface.hpp"
+
+namespace
+{
+using tier4_rtc_msgs::msg::Module;
+
+std::string to_string(const unique_identifier_msgs::msg::UUID & uuid)
+{
+  std::stringstream ss;
+  for (auto i = 0; i < 16; ++i) {
+    ss << std::hex << std::setfill('0') << std::setw(2) << +uuid.uuid[i];
+  }
+  return ss.str();
+}
+
+Module getModuleType(const std::string & module_name)
+{
+  Module module;
+  if (module_name == "blind_spot") {
+    module.type = Module::BLIND_SPOT;
+  } else if (module_name == "crosswalk") {
+    module.type = Module::CROSSWALK;
+  } else if (module_name == "detection_area") {
+    module.type = Module::DETECTION_AREA;
+  } else if (module_name == "intersection") {
+    module.type = Module::INTERSECTION;
+  } else if (module_name == "no_stopping_area") {
+    module.type = Module::NO_STOPPING_AREA;
+  } else if (module_name == "occlusion_spot") {
+    module.type = Module::OCCLUSION_SPOT;
+  } else if (module_name == "stop_line") {
+    module.type = Module::NONE;
+  } else if (module_name == "traffic_light") {
+    module.type = Module::TRAFFIC_LIGHT;
+  } else if (module_name == "virtual_traffic_light") {
+    module.type = Module::TRAFFIC_LIGHT;
+  } else if (module_name == "lane_change_left") {
+    module.type = Module::LANE_CHANGE_LEFT;
+  } else if (module_name == "lane_change_right") {
+    module.type = Module::LANE_CHANGE_RIGHT;
+  } else if (module_name == "avoidance_left") {
+    module.type = Module::AVOIDANCE_LEFT;
+  } else if (module_name == "avoidance_right") {
+    module.type = Module::AVOIDANCE_RIGHT;
+  } else if (module_name == "pull_over") {
+    module.type = Module::PULL_OVER;
+  } else if (module_name == "pull_out") {
+    module.type = Module::PULL_OUT;
```

Comment:
memo
TODO : In order to remove this, each scene module uses the ENUM defined by tier4_xx_msgs.
But this TODO doesn't have to do in this PR.